ruby/ql/test/library-tests/frameworks/core/Kernel.rb

```diff
@@ -87,4 +87,8 @@ def run
   end
 end
 
-UnknownModule.system("ls")
\ No newline at end of file
+UnknownModule.system("ls")
+
+method("exit").call
+public_method("exit").call
+singleton_method("foo").call
```

Comment:
Does this also recognise calls with receivers, such as `Foo.method("exit")`?

Reply:
Yes, good point 👍  I'll add more test cases for that